jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/UpdateFunctionConfigPanel.kt

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.lambda.upload
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.ValidationInfo
+import com.intellij.ui.IdeBorderFactory
+import software.aws.toolkits.resources.message
+import javax.swing.JPanel
+import javax.swing.JTextField
+
+class UpdateFunctionConfigPanel(private val project: Project) {
+    lateinit var content: JPanel
+        private set
```

Comment:
We should probably do private set's on components more often, cool

Reply:
Ya one class went from 50% coverage to almost 80%